db/schema.rb

```diff
@@ -95,7 +105,8 @@
     t.integer "property_id"
     t.decimal "rebate", precision: 8, scale: 2
     t.integer "batch_id"
-    t.boolean "completed", default: false
+    t.boolean "completed", default: true
```

Comment:
i think you're missing a migration -- probably your dev database it messed up by PRs. Consider dropping the database and regenning. You can make your users again with db:seed, and make some rebate forms with the frontend app.